src/components/Theme/style.css

```diff
@@ -29,19 +36,18 @@
 }
 
 .navigationBar{
+  height: @navbar-height;
+  padding: @navbar-all-padding;
   flex: 0 0 auto;
 }
 
 .content {
+  /* Content height is 100% - non content height (total height of footer + total height of navbar)  */
+  height: calc(100% - @non-content-height);
```

Comment:
nice :)
very minor, I would move the variable `@non-content-height:` assignment to this block, since it's only used here.

Reply:
Done